haiku-apps/simplebackup/simplebackup-2.0.1.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="Graphical interface to run the zip command"
+DESCRIPTION="This application makes it easy to create and update backups of \
+large amounts of data.  For example, you can create a backup of your home \
+directory and run this application regularly to make sure the backup is up to \
+date.  SimpleBackup also accepts command line options, so it is easily \
+scriptable and can be used with scheduling applications."
+HOMEPAGE="https://perelandra0x309.github.io/simplebackup"
+COPYRIGHT="2010-2016 Brian Hill"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/Perelandra0x309/simplebackup/archive/v${portVersion}.tar.gz"
+CHECKSUM_SHA256="463df905df35a74e81127d1c28d20a1f2c94ff1b89f6939abf081ecc70e60c62"
+SOURCE_DIR="simplebackup-${portVersion}"
```

Comment:
IMO, The SOURCE_DIR could really be dropped. The curlies in SOURCE_URI would only be needed if there were a space in the portVersion or some other special characters after them (?). If the {} aren't ususally use, we maybe shouldn't do so here either.